### ISSUE TITLE
New rule: Meta-refresh-no-exceptions

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3497]
The current meta-refresh rule doesn't actually check that the refresh exceeds the 20 hour limit, but once #3496 is resolved it will. We should have a version of this rule for WCAG level AAA that does not permit any delay. This would be a rule that is consistent with the [Meta element has no refresh delay (no exception)
](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/bisz58/proposed/) ACT rule.

**Comment by padmavemulapati** [comment]
Validated with latest develop branch code base,
`meta-refresh-no-exceptions` rule for pass and fail cases:
```
<head>
    <meta charset="utf8" />
    <title>Meta-refresh-no-exceptions fail 1</title>
    <meta http-equiv="refresh" content="72001: https://w3.org" />
    <meta http-equiv="refresh" content="90000; https://deque.com/">
    
    <link
      rel="stylesheet"
      type="text/css"
      href="/node_modules/mocha/mocha.css"
    />
```
<img width="1581" alt="image" src="https://user-images.githubusercontent.com/47409579/187353568-8c2062f1-36e3-4539-83cc-d018d4f75c5f.png">
